lib/sprite_animation.dart

```diff
@@ -136,19 +139,17 @@ class SpriteAnimation {
 
     final frames = jsonFrames.values.map((value) {
       final frameData = value['frame'];
-      final int x = frameData['x'];
-      final int y = frameData['y'];
-      final int width = frameData['w'];
-      final int height = frameData['h'];
+      final double x = frameData['x'];
+      final double y = frameData['y'];
+      final double width = frameData['w'];
+      final double height = frameData['h'];
```

Comment:
why this change? this might be a breaking change, we must see how aseprite stores data

Reply:
It is just JSON right, and since there only is one number type there it should be fine?

Reply:
there is only one number type in JSON but I am not sure how dart convert it to map. I am scared dart will throw an error (unlike js in dart int/double are distinct types). I think it might be safer to cast to int and call `toDouble()`, since it's a dynamic map. unless we can easily test it or there is an example and it works.